Makefile

```diff
@@ -125,11 +125,11 @@ flake8: ## Lints all Python files with flake8
 		| perl -F':\s+' -nE '$$F[1] =~ m/text\/x-python/ and say $$F[0]' \
 		| xargs flake8
 
-update-fedora-templates: assert-dom0 ## Upgrade Fedora 25 to Fedora 26 templates
-	sudo qubes-dom0-update qubes-template-fedora-26
-	sudo qubes-prefs default_template fedora-26
+update-fedora-templates: assert-dom0 ## Upgrade to Fedora 28 templates
+	sudo qubes-dom0-update qubes-template-fedora-28
```

Comment:
If the `qubes-template-fedora-28` is installed, this command will fail and return `Error: unable to find a match`. I think we need to check for the presence of the fedora 28 template beforehand (e.g., `rpm -qa | grep qubes-template-fedora-28`).